docs/api/enums.rst

```diff
@@ -242,6 +242,58 @@ of :class:`enum.Enum`.
         The system message denoting that an context menu command was executed.
 
         .. versionadded:: 2.0
+    .. attribute:: auto_moderation_action
+
+        The system message denoting an action by automod.
+
+        .. versionadded:: 2.3
```

Comment:
This should be 2.4?

Reply:
Nope, it was added earlier in Pycord-Development/pycord#1316 but not documented